py3status/modules/wifi.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+"""
+Display WiFi bit rate, quality, signal and SSID using iw.
+
+Configuration parameters:
+    - cache_timeout : Update interval in seconds (default: 5)
+    - device : Wireless device name (default: "wlan0")
+    - down_color : Output color when disconnected, possible values:
+      "good", "degraded", "bad" (default: "bad")
+      percent (default: false)
+    - signal_bad : Bad signal strength in percent (default: 29)
+    - signal_degraded : Degraded signal strength in percent
+      (default: 49)
+    - bitrate_bad : Bad bit rate in Mbit/s (default: 26)
+    - bitrate_degraded : Degraded bit rate in Mbit/s (default: 53)
+    - round_bitrate : If true, bitrate is rounded to the nearest whole number
+      (default: true)
+    - format_up : See placeholders below (default:
+      "W: {bitrate} {signal_percent} {ssid}").
+    - format_down : Output when disconnected (default: "down")
+
+Format of status string placeholders:
+    {bitrate} - Display bit rate
+    {signal_percent} - Display signal in percent
+    {signal_dbm} - Display signal in dBm
+    {ssid} - Display SSID
+
+Requires:
+    - iw
+
+@author Markus Weimar <mail@markusweimar.de>
+@license BSD
+"""
+
+import re
+import subprocess
+from time import time
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
```

Comment:
please sort the config params alphabetically (and please update the docstring respectfully)